src/web/src/components/NavBar/NavBar.tsx

```diff
@@ -66,19 +86,24 @@ const buttons: HeaderIcon[] = [
   },
 ];
 
-export default function DesktopHeader() {
+export default function NavBar() {
   const classes = useStyles();
+  const theme = useTheme();
+  const desktop = useMediaQuery(theme.breakpoints.up(1024));
 
   return (
     <AppBar className={classes.root}>
       <Toolbar className={classes.toolbar}>
-        <Link href="/" passHref>
-          <a className={classes.logoIcon}>
-            <Logo height={45} width={45} />
-          </a>
-        </Link>
-
-        <div className={classes.grow} />
+        {desktop && (
+          <>
+            <Link href="/" passHref>
+              <a className={classes.logoIcon}>
+                <Logo height={45} width={45} />
+              </a>
+            </Link>
+            <div className={classes.grow} />
+          </>
+        )}
 
         {buttons.map((button) => (
           <HeaderButton button={button} key={button.title} />
```

Comment:
Since we are using NavBar, let's use NavBarButton and NavBarIcon as well.
Sorry for only cath this now.

Reply:
Thanks for letting me know. All references to Header have been changed.